src/PrpShop/Main.cpp

```diff
@@ -91,6 +91,7 @@ PrpShopMain::PrpShopMain()
     fActions[kFileSaveAs]->setShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_S);
     fActions[kFileExit]->setShortcut(Qt::ALT + Qt::Key_F4);
     fActions[kWindowClose]->setShortcut(Qt::CTRL + Qt::Key_W);
+    fActions[kWindowClose]->setShortcutContext(Qt::WidgetWithChildrenShortcut);
```

Comment:
Out of curiosity, does `Qt::ApplicationShortcut` also work without needing to add the action to the `fBrowserTree`?

Reply:
It seems like it should, doesn't it?  It was the first option I tried as it appeared to be the correct context, but it appears to have no effect whatsoever.  Only `Qt::WidgetWithChildrenShortcut` works as expected.

Reply:
Huh, strange...  I wish it was better documented >.>